flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowLogDBService.java

```diff
@@ -85,15 +78,11 @@ public class FlowLogDBService implements FlowLogService {
     @Override
     public FlowLog save(FlowParameters flowParameters, String flowChainId, String key, Payload payload, Map<Object, Object> variables, Class<?> flowType,
             FlowState currentState) {
-        String payloadAsString = getSerializedString(payload);
-        String variablesJson = variables == null ? null : getSerializedString(variables);
         String payloadJackson = JsonUtil.writeValueAsStringSilent(payload);
-        JsonUtil.checkReadability(payloadJackson, Payload.class);
         String variablesJackson = TypedJsonUtil.writeValueAsStringSilent(variables);
-        TypedJsonUtil.checkReadability(variablesJackson, Map.class);
 
         FlowLog flowLog = new FlowLog(payload.getResourceId(), flowParameters.getFlowId(), flowChainId, flowParameters.getFlowTriggerUserCrn(), key,
-                payloadAsString, payloadJackson, ClassValue.of(payload.getClass()), variablesJson, variablesJackson,
+                "", payloadJackson, ClassValue.of(payload.getClass()), "{}", variablesJackson,
```

Comment:
Not sure it matters but why not use `"{}"` for the legacy `payloadAsString` as well?

BTW, are there any future plans for dropping the legacy json.io fields from the entity tables? If so, the value stored here does not really make a difference. (Unless we want to keep backward compatibility with very old CB versions, which is unlikely.)

Reply:
Got rid of these fields